src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -170,10 +182,13 @@ private BoundAttribute BindAttributeCore(AttributeSyntax node, NamedTypeSymbol a
             ImmutableArray<BoundExpression> boundNamedArguments = analyzedArguments.NamedArguments;
             constructorArguments.Free();
 
-            return new BoundAttribute(node, attributeConstructor, boundConstructorArguments, boundConstructorArgumentNamesOpt,
+            var attribute = new BoundAttribute(node, attributeConstructor, boundConstructorArguments, boundConstructorArgumentNamesOpt, argsToParamsOpt, expanded,
                 boundNamedArguments, resultKind, attributeType, hasErrors: resultKind != LookupResultKind.Viable);
-        }
 
+            NullableWalker.AnalyzeIfNeeded(Compilation, attribute, diagnostics);
```

Comment:
>NullableWalker.AnalyzeIfNeeded(Compilation, attribute, diagnostics); [](start = 12, length = 68)

I think this call should be moved to the ```internal CSharpAttributeData GetAttribute(AttributeSyntax node, NamedTypeSymbol boundAttributeType, DiagnosticBag diagnostics)``` so that we wouldn't do this analysis when SemanticModel binds the attribute.